index.html

```diff
@@ -26,5 +27,121 @@ <h2 class="intro-section-item" id="intro-text2">I’am a Software Developer</h2>
       <li><a href="" class="icon-element" type="button"><img src="src/m.png" alt="m"></a></li>
     </ul>
   </section>
+
+  <section class="recent-works">
+    <h1 class="works">My Recent Works</h1>
```

Comment:
I suggest you change this to any other heading `<hx> from <h2> to <h6>`. Your document should only have one `<h1>` element. Your browsers uses this for seo optimization and also for your site to be used by screen readers. Read [here](https://ahrefs.com/blog/h1-tag/#:~:text=skim%20and%20digest.-,H1%20tags%20improve%20accessibility,title%20be%20an%20H1%20tag.) for more on this